plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/index.tsx

```diff
@@ -126,16 +126,18 @@ export const TotalsShipping = ( {
 							<ShippingVia
 								selectedShippingRates={ selectedShippingRates }
 							/>
-							<ShippingAddress
-								shippingAddress={ shippingAddress }
-								showCalculator={ showCalculator }
-								isShippingCalculatorOpen={
-									isShippingCalculatorOpen
-								}
-								setIsShippingCalculatorOpen={
-									setIsShippingCalculatorOpen
-								}
-							/>
+							{ showCalculator && (
+								<ShippingAddress
+									shippingAddress={ shippingAddress }
+									showCalculator={ showCalculator }
```

Comment:
Why are we passing showCalculator but still gating by it?

Reply:
@senadir We will be removing it in the next PR:https://github.com/woocommerce/woocommerce/pull/47953/files#diff-88f77c3a9158a573d19d0af92927173bc105e708c0e5a167993d1ebc924f1342L132. I kept it so it doesn't break any functionality of this PR.